xml/src/main/resources/ehcache-core.xsd

```diff
@@ -382,7 +381,9 @@
                   <xs:attribute name="coalesce" type="xs:boolean" default="false" use="optional"/>
                 </xs:complexType>
               </xs:element>
-              <xs:element name="non-batching"/>
+              <xs:element name="non-batching">
```

Comment:
Huh?

Reply:
A trick to force JAXB to generate a type for that element to ease my unparsing. And it turned out to be better than the previous definition which allowed any random content and attributes. This is more restrictive.